generate_directory_indexes.py

```diff
@@ -208,6 +213,16 @@ def is_excluded_file(file_name):
 
     return file_name in excluded_file_names
 
+def is_excluded_path(path):
+    exclude = False
+    # for the ability to match more exactly, get rid of base_path
+    path = path.replace(configuration.base_path, '')
+    if configuration.exclude_path is not None:
+        # similar to configuration[exclude_path].select { |x| x =~ /^\/#{path}/ }.any? in ruby
+        exclude = exclude or len(filter(lambda x: re.match('/{path}'.format(path=x), path), configuration.exclude_path)) != 0
```

Comment:
Probably something silly that I'm missing but why not this:?

```python
if configuration.exclude_path is not None:
    return len(filter(lambda x: re.match('/{path}'.format(path=x), path), configuration.exclude_path)) != 0
```

Reply:
ah no I think this is leftover cruft from an earlier implementation where there were more things I checked, will clean up

Reply:
done!